### ISSUE TITLE
SCSS padding and margin values using $variables are zeroed out

### ISSUE THREAD

**Issue by pamtbaau** [Glavin001/atom-beautify#697]
After beautifying scss file, values for margin/padding that use $variables are being zeroed out.

```
p {
    margin: 0 0 ($font-size-base * $line-height-base) 0;
    padding: 0 0 ($font-size-base * $line-height-base) 0;
}
```

SCSS result after save:

```
p {
    margin : 0;
    padding: 0;
}
```

**Comment by prettydiff** [comment]
I am looking into this now.

**Comment by prettydiff** [comment]
Resolved but not yet published: https://github.com/prettydiff/prettydiff/issues/231

**Comment by prettydiff** [comment]
Published in v1.16.1.

**Comment by prettydiff** [comment]
Fixed in the latest version of Pretty Diff which is now in Atom Beautify.
